src/frontend/components/sidebar/sidebar.css

```diff
@@ -36,6 +36,7 @@
         background-color: var(--color-background-accent);
         height:87%;
         box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
```

Comment:
you can use variables for the rgba color

Reply:
i will try